common/djangoapps/student/models.py

```diff
@@ -2402,6 +2416,18 @@ def _key(self):
         """
         return (self.role, self.org, self.course_id, self.user_id)
 
+    @classmethod
+    def access_roles_in_course(cls, course_key):
+        """
+        Returns all users that have a course access role in a given course.
```

Comment:
It doesn't return `all users` but all `CourseAccessRoles` for a given course, with user information prefetched.

```suggestion
        Returns all CourseAccessRole for a given course and prefetches user information.
```